From d967caa988eaabd9e84c82879e2f21bd33b952a7 Mon Sep 17 00:00:00 2001
From: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
Date: Sun, 29 May 2016 23:39:06 -0700
Subject: [PATCH 5/9] fstrans: Remove unused foo() function

The point of it seems to have been to silence an unused function warning, but
there's no point if we're just transitioning that to another unused function
warning.

src/fc/fstrans.c:32:20: warning: unused function 'foo' [-Wunused-function]
static inline void foo(void) { (void) is_numeric("a"); }
                   ^
1 warning generated.

Signed-off-by: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
Reviewed-by: Keith Packard <keithp@keithp.com>
---
 src/fc/fstrans.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/fc/fstrans.c b/src/fc/fstrans.c
index 66bc978..902ef36 100644
--- a/src/fc/fstrans.c
+++ b/src/fc/fstrans.c
@@ -28,5 +28,3 @@
 #define FONT_t
 #define TRANS_CLIENT
 #include <X11/Xtrans/transport.c>
-/* inhibit warning about is_numeric */
-static inline void foo(void) { (void) is_numeric("a"); }
-- 
2.9.3

